docs/deployment/config_descriptions.md

```diff
@@ -0,0 +1,77 @@
+---
+id: config_descriptions
+title: Configuration Descriptions
+hide_title: true
+---
+
+# IAM Configuration
+
+### iam-deployment-name:
+
+The name (e.g. dev) of the the 'deployment' - this key tells kes which configuration set (in addition to the default values) to use when creating the cloud formation template[^1]
+
+### prefix:
+
+This value will prefix CloudFormation-created IAM resources and permissions.
+
+**The cumulus stack name must start with `<prefix>`** [^2]
```

Comment:
I feel like it's a little confusing to have this note under `prefix` instead of `stackName`, but maybe that's just me

Reply:
I moved it.